packages/react-scripts/template/src/App.js

```diff
@@ -13,22 +13,24 @@ const I18nPage = React.lazy(() => import('./components/example/I18nPage'))
 
 function App() {
   return (
-    <ErrorBoundary>
-      <Suspense fallback={<HomePageSkeleton />}>
-        <Tabs>
-          <Tab title="Home" to="./" />
-          <Tab title="User Info" to="/user" />
-          <Tab title="I18n" to="/i18n" />
-        </Tabs>
+    <AppLayout fullWidth>
```

Comment:
how does this compare to `<Root fullWidth>`?

Reply:
This is different than the instructions in zion-ui docs. Those docs said to replace `<Root fullWidth>` in index.js with `<Root><AppLayout fullWidth>` .  If this is more correct, those docs should be updated as well.

Reply:
Yes, docs need to be updated.